project/plugins.sbt

```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 addSbtPlugin("com.jsuereth"      % "sbt-pgp"       % "2.0.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"    % "5.2.0")
 addSbtPlugin("com.github.gseitz" % "sbt-release"   % "1.0.12")
+addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
```

Comment:
The plugin is there (2 lives above), but probably it is not automatically invoked on build steps.

Reply:
Just checked the plugin docs. You can add `.enablePlugins(AutomateHeaderPlugin)` to build.sbt live above and headers will be created automatically:

```scala
lazy val hats = (project in file("."))
  .settings(
    name := "spark-hats",
    printSparkVersion := {
      val log = streams.value.log
      log.info(s"Building with Spark $sparkVersion")
      sparkVersion
    },
    (Compile / compile) := ((Compile / compile) dependsOn printSparkVersion).value,
    libraryDependencies ++= SparkHatsDependencies :+ getScalaDependency(scalaVersion.value),
    releasePublishArtifactsAction := PgpKeys.publishSigned.value,
    Test / fork := true
  ).enablePlugins(AutomateHeaderPlugin)
```

Reply:
Nice. Fixed